js/views/modals/orderDetail/summaryTab/OrderDetails.js

```diff
@@ -59,6 +60,50 @@ export default class extends BaseVw {
     return !!this.model.get('buyerOrder').payment.moderator;
   }
 
+  /**
```

Comment:
Yikes! This order data structure is so complicated in regards to variants... this whole function was needed just to extract the SKU of the ordered product.